specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/SAPVirtualInstance.json

```diff
@@ -314,17 +302,17 @@
           "Install SAP Software on Distributed System": {
             "$ref": "./examples/sapvirtualinstances/SAPVirtualInstances_Install_Distributed.json"
           },
-          "Create Infrastructure with Disk and OS configurations for Single Server System (Recommended)": {
+          "Create Infrastructure with Disk and OS configurations for Single Server System": {
             "$ref": "./examples/sapvirtualinstances/SAPVirtualInstances_Create_DiskDetails_SingleServer.json"
           },
-          "Create Infrastructure with Disk and OS configuration for Distributed System (Recommended)": {
+          "Create Infrastructure with Disk and OS configuration for Distributed System": {
             "$ref": "./examples/sapvirtualinstances/SAPVirtualInstances_Create_DiskDetails_Distributed.json"
           },
-          "Create Infrastructure with Disk and OS configuration for HA System with Availability Set (Recommended)": {
```

Comment:
Can you please retain the part about Availability set. Need not mention backup disk configured. It's enough to mention where it's skipped, I believe.